services/api/src/Handlers.ts

```diff
@@ -156,6 +156,22 @@ export function search(db: Database, req: express.Request, res: express.Response
     requirespenpaper: body.requirespenpaper,
     text: body.text,
   };
+
+  if (params.partition === PUBLIC_PARTITION && res.locals.id) {
+    // Also search for private quests by this user
+    const privateParams = {...params, owner: res.locals.id, partition: PRIVATE_PARTITION};
+    return Promise.all([
+      doSearch(db, res.locals.id, params),
+      doSearch(db, res.locals.id, privateParams),
+    ]).then((results) => {
+      res.status((results[0].error) ? 500 : 200).end(JSON.stringify({
+        error: results[0].error,
+        hasMore: results[0].hasMore,
+        quests: [...results[1].quests, ...results[0].quests],
```

Comment:
Is this guaranteed to return private then public?

Reply:
yep